### DISCUSSION TITLE
State vuex

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#541] [Q&A] [answered]
Buenas, disculpen estaba poniendo los a adornos, sabores, precios y cantidad

La verdad no se si esta bien, ¿Me podrian decir si esta bien o mal? Por favor

import { createStore } from 'vuex'

export default createStore({
  state: {
    sabores: {
      s1: 'Vainilla',
      s2: 'Chocolate',
      s3: 'Fresa',
      s4: 'Limón',
      s5: 'Oreo',
      s6: 'Café',
      s7: 'Moka',
      s8: 'Dulce de leche',
      s9: 'Coco',
      s10: 'Tres leches'
    },
    preciogr: {
      gr123: '$150 MXN',
      gr4: '$200 MXN',
      gr567: '$250 MXN',
      gr89: '$300 MXN',
      gr10: '$350 MXN',
    },
    preciomed: {
      med123: '$100 MXN',
      med4: '$150 MXN',
      med567: '$200 MXN',
      med89: '$250 MXN',
      med10: '$300 MXN'
    },
    precioch: {
      ch123: '$50 MXN',
      med4: '$100 MXN',
      med567: '$150 MXN',
      med89: '$200 MXN',
      med10: '$350 MXN'
    },
    scantidad: {
      sc1: '50',
      sc2: '30',
      sc3: '20',
      sc4: '15',
      sc5: '10',
      sc6: '8',
      sc7: '25',
      sc8: '40',
      sc9: '38',
      sc10: '17'
    },
    adornos: {
      a1: 'Cobertura de chocolate',
      a2: 'Fondant de base de malvavisco',
      a3: 'Fondant de chocolate',
      a4: 'Fondant de chocolate blanco',
      a5: 'Salsa de caramelo',
      a6: 'Betún de vainilla',
      a7: 'Betún de chocolate',
      a8: 'Betún de fresa',
      a9: 'Escarchado',
      a10: 'Glaseado',
      a11: 'Frutas', 
      a12: 'Chipas',
    },
    precioa: {
      pa1: '$50 MXN',
      pa234: '$100 MXN',
      pa5: '$70 MXN',
      pa9: '$60 MXN',
      pa10: '$80 MXN',
      pa11: '$20 MXN',
      pa12: '$10 MXN'
    },
    acantidad: {
      ac1: '30',
      ac2: '20',
      ac3: '17',
      ac4: '25',
      ac5: '12',
      ac6: '10',
      ac7: '5',
      ac8: '33',
      ac9: '35',
      ac10: '37',
      ac11: '8',
      ac12: '18'
     }
  },
  getters: {
  },
  mutations: {
  },
  actions: {
  },
  modules: {
  }
})

**Comment by doguedogue** [comment, marked as answer]
Genial, vas muy avanzada.

Yo te recomendaría y es como consejo aunque no necesario, usar los agrupadores de elementos: arreglos []
Y para los elementos objetos {}

Ya que estos se pueden barrer con ciclos for, si es que así lo quieres, imagina que la información viene de una API, así como los pokemons (o pokemones, cuál es el plural?)

**Reply by GloriaCoralCerecedo** [reply]
Gracias!, la verdad es que apenas estoy empezando pero estoy haciendo poco a poco lo que voy entendiendo.

Y si tambien pensé con los [], igual intente poner un objeto dentro del otro pero me imaginé que así estaría mejor para entender.
